pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -69,6 +69,10 @@ var (
 	AntreaInfoElementsIPv6 = append(antreaInfoElementsCommon, []string{"destinationClusterIPv6"}...)
 )
 
+const (
+	FlowAggregatorDNSName = "flow-aggregator.flow-aggregator.svc.cluster.local"
```

Comment:
my recommendation is to use `"flow-aggregator.flow-aggregator.svc"`. IIRC the cluster's domain name will automatically be added to the Pod's `/etc/resolv.conf`. This allows this default to work even when the cluster's domain name is not the default (`cluster.local`).

Reply:
Changed it. Thanks.